app/models/git/exercise.rb

```diff
@@ -285,5 +288,8 @@ def commit = repo.lookup_commit(git_sha)
 
     memoize
     def track = Track.new(repo:)
+
+    SKIP_TEST_RUNS_MAGIC_MARKER = "[skip tests]".freeze
```

Comment:
I've tried to make it use a similar syntax to GitHub's magic markers (e.g. `[skip ci]`). If we think this is a good plan, we could consider having this be a list of magic markers (e.g. `[skip test runs]` or `[tests skip]`).

Reply:
I like this. I'd like to use `exercism` in the message though.

`[exercism: skip test reruns]` maybe?